packages/chart/src/ChartUtils.ts

```diff
@@ -757,13 +805,60 @@ class ChartUtils {
       seriesData.marker.color = shapeColor;
     }
 
+    if (shape != null && shape.length > 0) {
+      try {
+        seriesData.marker.symbol = ChartUtils.getMarkerSymbol(shape);
+      } catch (e) {
+        log.warn('Unable to handle shape', shape, ':', e);
+      }
+    }
+
+    if (shapeSize != null) {
+      seriesData.marker.size = shapeSize * ChartUtils.DEFAULT_MARKER_SIZE;
+    }
+
     // Skipping pie charts
     // Pie slice visibility is configured in chart layout instead of series data
     if (seriesVisibility != null && plotStyle !== dh.plot.SeriesPlotStyle.PIE) {
       seriesData.visible = seriesVisibility;
     }
   }
 
+  /**
+   * Get the marker symbol for the provided Deephaven shape
+   * Deephaven shapes: https://deephaven.io/enterprise/docs/plotting/visual-formatting/#point-formatting
+   * Plotly shapes: https://plotly.com/javascript/reference/scattergl/#scattergl-marker-symbol
+   * Table of plotly shapes: https://plotly.com/python/marker-style/
+   * @param shape Shape to get the marker symbol for
+   */
+  static getMarkerSymbol(shape: string): MarkerSymbol {
+    switch (shape) {
+      case 'SQUARE':
+        return 'square';
+      case 'CIRCLE':
+        return 'circle';
+      case 'DIAMOND':
+        return 'diamond';
+      case 'UP_TRIANGLE':
+        return 'triangle-up';
+      case 'DOWN_TRIANGLE':
+        return 'triangle-down';
+      case 'RIGHT_TRIANGLE':
+        return 'triangle-right';
+      case 'LEFT_TRIANGLE':
+        return 'triangle-left';
+      // There don't seem to be any plotly equivalents for ellipse or rectangles
+      // Rectangles could be `line-ew`, `line-ns`, or `hourglass` and `bowtie` instead?
+      // Ellipse could be `asterisk` or `diamond-wide` instead?
+      // Just throw an error for now, we've already got a bunch of types.
```

Comment:
Should open a ticket for these?

Reply:
https://github.com/deephaven/deephaven-core/issues/3253